app/scripts/loadout/dimLoadoutPopup.directive.js

```diff
@@ -120,7 +119,9 @@
         name: name,
         items: _(items)
           .chain()
-          .select('equipped')
+          .select(function(item) {
```

Comment:
we lost `.select('equipped')` here (per https://github.com/DestinyItemManager/DIM/pull/507#issuecomment-217879350)